### ISSUE TITLE
Change send-data service data to strictly be an object (page 2 & page 3)

### ISSUE THREAD

**Issue by amj100** [bliechty/trivia#14]
***I think we should use this issues tab to make checklists/issues for what we need to get done that way we can easily visualize and complete the things we need to complete.

Anyways I don't care too much about what data we use but for page 3 it seems that I mainly need two arrays: an array of the users and an array of the questions. Right now send-data service data only has the questions rn but when we get the users functionality set up we might need to send the data with an object.

TL;DR we should change the send-data service to be an object that holds the information like this:

data: object = {
          questions: [],
          users: []
}

**Comment by amj100** [comment]
fixed by #17
